deploy/kubernetes/helm/che/charts/che-keycloak/templates/deployment.yaml

```diff
@@ -45,7 +45,19 @@ spec:
           value: admin
         - name: CHE_HOST
           value: {{ template "cheHost" . }}
+        - name: ROUTING_SUFFIX
```

Comment:
Is it possible to use the [Che HostHelper template](https://github.com/eclipse/che/blob/master/deploy/kubernetes/helm/che/templates/_hostHelper.tpl) instead of passing the ingress domain, namespace, and constructing the hostname in the configuration script?
This will be more consistent with the way we configure ingresses, config maps, etc.

Reply:
Yes, i have thought about it, but I will then need to change the entrypoint script that expects PROTOCOL and ROUTING_SUFFIX envs.